client/src/assets/components/toast/ErrorToast.jsx

```diff
@@ -0,0 +1,35 @@
+import { motion, AnimatePresence } from "framer-motion";
+import { Alert } from "react-daisyui";
+
+const ErrorToast = ({ message, show, icon, iconBgColor, status, onClose }) => {
+  return (
+    <AnimatePresence>
+      {show && (
+        <motion.div
+          initial={{ opacity: 0, y: 50 }}
+          animate={{ opacity: 1, y: 0 }}
+          exit={{ opacity: 0, y: -50 }}
+          transition={{ duration: 0.4 }}
+          className="fixed top-5  transform -translate-x-1/2 z-50 w-[55%] "
+        >
+          <Alert
+            className={`shadow-lg relative pr-2 rounded-[8px]  bg-[#e4b3b3eb]`}
```

Comment:
I LOVE the way you designed these toast pop ups!
The only change I request is that instead of the custom bg, you use "alert alert-error," so it matches the scheme of our theme, please.